lighthouse-core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -23,9 +23,21 @@ writeFileSync(filename, cleanAndFormatLHR(data), 'utf8');
  * @return {string}
  */
 function cleanAndFormatLHR(lhrString) {
+  /** @type {LH.Result} */
   const lhr = JSON.parse(lhrString);
-  delete lhr.configSettings.auditMode;
-  delete lhr.timing;
+
+  // Set to predictable and typical value
+  lhr.configSettings.auditMode = false;
+
+  // Set timing values, which change from run-to-run, to predicable values
+  lhr.timing.total = 12345.6789;
+  lhr.timing.entries.forEach((entry, i) => {
+    // @ts-ignore
+    entry.duration = 100;
```

Comment:
why are these ignored? `readonly`? Maybe we don't want them to descend from `PerformanceEntry` :)

Should put a reason why it's being ignored. e.g. `// @ts-ignore - write to readonly property` or whatever